packages/fxa-auth-server/lib/routes/recovery-key.js

```diff
@@ -296,8 +296,10 @@ module.exports = (
       options: {
         ...RECOVERY_KEY_DOCS.RECOVERYKEY_EXISTS_POST,
         auth: {
-          mode: 'optional',
-          strategy: 'sessionToken',
+          strategies: [
+            'multiStrategySessionToken',
+            'multiStrategyPasswordForgotToken',
+          ],
         },
         validate: {
           payload: isA.object({
```

Comment:
We could remove this email here since we're no longer allowing unauthenticated use of this endpoint?

Reply:
We could, yes.  I don't know how much clean up it would require.  So, I could either file a follow-up jira or I can clean it up as part of this patch.  Since this PR is blocking you, what do you think?

Reply:
Maybe it is scope creep since it doesn't technically cause problems to keep email as an optional payload... Could perhaps be noted in the backbone reset password cleanup ticket (`FXA-8267`) since that's the only remaining place (that I can see) that attempts to check for recovery key with an email.

Reply:
Follow up is probably fine. I don't see any risks with removing the `email` value since we should be the only ones using it on our frontends. I don't believe others are using the email value in the api, or this api at all lol.

Reply:
> I don't see any risks with removing the email value ...
Oh yeah, I'm not worried about breaking anything.  It's just a matter of how much delay we want to tolerate to get the code as clean as we'd like.

Reply:
> Could perhaps be noted in the backbone reset password cleanup ticket (FXA-8267) since that's the only remaining place ...

I was more concerned with the auth-client still accepting a payload for the email (and whatever might be using it).  I assume the Backbone stuff will be deleted wholesale at some point anyway.